src/gradata/daemon.py

```
@@ -483,6 +720,74 @@ def _cleanup(self) -> None:
+            return
+
+        match = _re.search(r'telemetry_last_sent\s*=\s*"([^"]+)"', config_text)
+        if match:
+            try:
+                last_sent = datetime.fromisoformat(match.group(1))
+                if (datetime.now(timezone.utc) - last_sent).total_seconds() < 86400:
+                    return
+            except ValueError:
+                pass
+
+        def _send():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_send`

Add return type annotation: `None`

(ANN202)